examples/cql_collections.rs

```diff
@@ -0,0 +1,65 @@
+use anyhow::Result;
+use scylla::transport::session::{IntoTypedRows, Session};
+use std::collections::HashMap;
+use std::env;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Create connection
+    let uri = env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+
+    println!("Connecting to {} ...", uri);
+
+    let session = Session::connect(uri, None).await?;
+    session.refresh_topology().await?;
+
+    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await?;
+
+    // Create the table
+    session
+        .query(
+            "CREATE TABLE IF NOT EXISTS teams (id int, team map<int, text>, primary key id)",
+            &[],
+        )
+        .await?;
+
+    // Send some data
+    session
+        .query(
+            "INSERT INTO teams (id, team) VALUES (1, {1 : 'John', 2 : 'Jessica'})",
+            &[],
+        )
+        .await?;
+
+    session
+        .query(
+            "INSERT INTO teams (id, team) VALUES (2, {1 : 'Ginny', 2 : 'Cho'})",
+            &[],
+        )
+        .await?;
```

Comment:
The test does not check that serializing maps work - but it should. It should include something like this:

```rust
let teams: HashMap<i32, String> = vec![
    (1, "Ginny".to_owned()),
    (2, "Cho".to_owned()),
].into_iter().collect();

session
    .query(
        "INSERT INTO teams (id, team) VALUES (2, ?)",
        &[teams],
    )
    .await?;
```

There should also be tests like this for lists, sets and tuples.

Reply:
Done.